sipn_reanalysis_ingest/util/url.py

```diff
@@ -8,21 +8,38 @@
 from sipn_reanalysis_ingest.util.product_type import cfsr_product_type_prefix
 
 
-def _cfsr_5day_tar_baseurl(*, window_start: dt.date) -> str:
+def _cfsr_day_tar_baseurl(*, window_start: dt.date) -> str:
```

Comment:
For consistency with other function names, what do you think about `daily`? (NOTE: accepting this suggestion won't update the callers of this function)

```suggestion
def _cfsr_daily_tar_baseurl(*, date: dt.date) -> str:
```

Reply:
I like that suggestion! I will update everywhere else once I commit here.